test/fixtures/plugins/fixture4-plugin/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pluginModule": {
+    "name": "fixture-plugin",
```

Comment:
We should give proper definitions for plugins instead of generic fixture-plugin, fixture2-plugin etc.